.chloggen/collectdreceiver-migrate-from-opensus-to-pdata.yaml

```diff
@@ -9,7 +9,7 @@ change_type: enhancement
 component: receiver/collectdreceiver
```

Comment:
the format of changlog seems not right? Do you generate this file by `make chlog-new`?

Reply:
btw, I think this pr do not need changelog, since there is no changes in terms of the users

Reply:
Ah, it looks like Zeno updated his existing changelog entry. Every PR needs to have its own changelog or no changelog as @fatsheep9146 explains. I agree this PR doesn't need a changelog.